app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
 
   belongs_to :draw
+  belongs_to :group
```

Comment:
hmm, I'm not sure. 

At the moment, leaders are not included in the members of a draw, so there's no conflict. Do you think it will help us in the long run to duplicate leaders in the members array? I'm still not sure that we would need the `has_one` because it would always be the same id as the `belongs_to`.